draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -249,17 +249,24 @@ degree of certainty of those answers.
 
 Congestion control algorithms without empirical evidence of Internet-scale
 deployment SHOULD seek Experimental status.
+Specifications published as Experimental ought to explain the reason for
+the status and what further information would be required to progress to
+standards track. For example, section 12. of  RFC6928 provides
+“Usage and Deployment Recommendations” that describe the experiments
+expected by the TCPM working group. Section 4 of RFC4614 provides other
```

Comment:
4614 has been obsoleted by 7414, but the later document is not as careful about formal Experimental status as the earlier one, so maybe your choice to reference 4614 is right.

But maybe this should be a reference?